test/WebApiShimFw46.FunctionalTests/Controllers/ValuesController.cs

```diff
@@ -20,6 +20,10 @@ public IEnumerable<string> Get()
         [HttpGet("{id}")]
         public string Get(int id)
         {
+            if (id == 42)
```

Comment:
This is a very profound id value :grinning:. Existential issues aside, I think tests would be cleaner with a separate method, similar to the `Exception` method in the MVC functional test.